sklearn/discriminant_analysis.py

```diff
@@ -71,14 +79,14 @@ def _cov(X, shrinkage=None, covariance_estimator=None):
                 s = sc.scale_[:, np.newaxis] * s * sc.scale_[np.newaxis, :]
             elif shrinkage == "empirical":
                 s = empirical_covariance(X)
-            else:
-                raise ValueError("unknown shrinkage parameter")
+            # else:
+            #     raise ValueError("unknown shrinkage parameter")
```

Comment:
No need for retaining these as comments I believe. Similarly to the other old validation checks.

```suggestion
```